week07/day03/Hello DI project/Views/Home/index.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿<!DOCTYPE html>
+
+<html>
+<head>
+    <meta name="viewport" content="width=device-width" />
+    <title>@ViewBag.Title</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <ul>
+                <li><a href="~/useful/colored/">COLORED</a></li>
```

Comment:
I'd avoid using the `~`. Most of the time it's best to use absolute urls `/useful/colored`, sometimes it's ok to use relative `./colored`.

Reply:
Better yet in case of Razor: use `HTML.ActionLink`